frontend/src/views/OrderView.vue

```diff
@@ -217,6 +217,12 @@ onMounted(async () => {
               :inputStyle="{ width: '100%' }"
             />
           </div>
+          <div v-if="isEditing">
+            <p class="font-bold">Unteraufträge</p>
+            <router-link :to="`${ROUTES.ORDER.path}/${route.params.id}/edit/offer/new`">
+              <Button label="Create offer"></Button>
+            </router-link>
+          </div>
```

Comment:
Does it make sense to have the offer-view as a separate page?

When the user wants to navigate from one invoice to the offer to the payment reminder he will need to switch pages a lot, right?

What do you think about keeping the offer-view as a component inside the order-view?